en/debug/mavlink_shell.md

```diff
@@ -0,0 +1,39 @@
+# MAVLink Shell
+
+The MAVLink Shell is an *NSH console* that can be accessed via MAVLink over serial (USB/Telemetry) or WiFi (UDP/TCP) links (in particular, on NuttX-based systems like: Pixhawk, Pixracer, etc.).
+
+The shell can be used for running commands and modules (and displaying the output of modules that it starts). 
+It cannot display the output of tasks running on the work queue (or modules that it does not start).
```

Comment:
You can do that now with `dmesg -f &`

Reply:
Fixed (as per https://github.com/PX4/Devguide/pull/953/files#r380466900). Can you sanity check the change.